mlir/lib/Target/LLVMIR/DebugTranslation.h

```diff
@@ -89,6 +89,7 @@ class DebugTranslation {
   llvm::DISubrange *translateImpl(DISubrangeAttr attr);
   llvm::DISubroutineType *translateImpl(DISubroutineTypeAttr attr);
   llvm::DIType *translateImpl(DITypeAttr attr);
+  llvm::DIImportedEntity *translate(DIImportedEntityAttr attr, llvm::DIScope *);
```

Comment:
Nit: This needs a comment, given that it does something non-obvious.

Reply:
Done.